blog/2023-01-20-Apache-Pulsar-2-11-0.md

```diff
@@ -10,7 +10,9 @@ The Apache Pulsar community releases version 2.11! 61 contributors provided feat
 
 ## Highlights of the 2.11 release
 
-* Support chunking with shared subscriptions ([PR-16202](https://github.com/apache/pulsar/pull/16202)) 
+This most significant compatibility change is that, starting from 2.11.0, the minimal version of JDK on the server side (broker, functions, connectors) is required to be 17. You can read [PIP-156](https://github.com/apache/pulsar/pull/15264) for more details.
```

Comment:
I think this paragraph breaks the relationship between the title and the following bullets. 
Maybe we can either make it a note under those bullets, or add one more sentence to introduce the following bullets, like "The highlighted features introduced in the 2.11 release include".

Reply:
Thanks for your suggestion! Updating...

Reply:
Updated. Please take another look ）